test/check-machines-multi-host-consoles

```diff
@@ -68,7 +68,9 @@ class TestMultiMachineVNC(VirtualMachinesCase):
         self.goToVmPage(name)
         b.wait_in_text(f"#vm-{name}-system-state", "Running")
 
-        b.add_machine(m2_host, known_host=True, password=None)
+        # Cockpit 329 warns the user before adding a remote host
+        cockpit_version = m.execute("cockpit-bridge --version | head -n 1 | awk '{ print $2 }'").strip()
+        b.add_machine(m2_host, known_host=True, password=None, expect_warning=cockpit_version > "328")
```

Comment:
You can use the `self.system_before(329)` helper for that.

Reply:
Nice!